### DISCUSSION TITLE
Remove Bebpack builder

### DISCUSSION THREAD

**Discussion by martinszeltins** [nuxt/nuxt#25585] [Ideas]
It seems that the whole frontend ecosystem (except for Next.js) has moved to Vite, leaving Webpack behind. It seems reasonable to ditch the Webpack builder altogether to avoid fragmentation and legacy code. Also, a good chunk of documentation could be removed (sections about webpack).

Is there any good reason to keep Webpack as a build option in Nuxt 3?

**Comment by AloisSeckar** [comment]
I would say at least to ease up migrating from Nuxt 2. Because this would be another level of complexity. I guess there are still a lot of projects in the wild that weren't migrated yet.
